Library/UIRefreshControl+StartRefreshing.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+import UIKit
+
+public extension UIRefreshControl {
+  func ksr_beginRefreshing() {
```

Comment:
Fixes an issue that prevented the refresh control from animating when the view appears. I think we have this same issue on `DiscoveryPageViewController`, this might be a simpler way of solving that 🤔